andrusiv-lesson-2/script.js

```diff
@@ -0,0 +1,19 @@
+let myName = "Oleh";
+console.log(myName);
+let mySurname = "Andrusiv";
+console.log(mySurname);
+let myCity = "Kyiv"; // become my native after more than 10 years of living there

```

Comment:
Try to use `const` where it is needed. Your name, for example, will not change, I guess :)

Reply:
Fixed in the new commit